src/book/03-guides/01-main-codebase/06-camera-calibration.mdx

```diff
@@ -28,15 +28,14 @@ The calibration requires two steps, creating the `nbs` recording and using it to
 
 5.  Check the focus of the lenses using the NUsight vision tab. If it is not sharp, focus the camera. This involves pulling the camera out from the head, still plugged in, and loosening the three grub screws. Twist until the camera is focused.
 
-6.  Check the [transformation matrix](/system/foundations/mathematics#homogeneous-transformations) from the left camera to the pitch motor (Hpc) is correct in the `Left.yaml` file. This should be the transformation matrix from the camera to the end of the rigid body, i.e. the pitch motor as used in forward kinematics. Currently this is 69.952mm in $x$, 33.795mm in $y$, 64.88mm in $z$, and a positive $y$-axis rotation of $2^\circ$. In SI units the matrix is
+6.  Verify that the extrinsics look correct in NUsight. The extrinsics are the transform from the last neck motor to the camera. This is retrieved from the robot's URDF model, but differences in the real robots may result in required offsets. Offset values for both pitch and roll are in the camera configuration files. The offsets are in radians.
```

Comment:
I think this definition might be better in either the mathematics or glossary pages rather than inline here

Reply:
Which part?

Reply:
The point of this line is to explain what the user needs to do (make sure the offsets are tuned so that the extrinsics are correct). I don't think the rest of this step makes a lot of sense without an explanation of what the user will be doing and where to do it. There could be a glossary entry on camera extrinsics and intrinsics but I don't think that would replace this.